metaflow/plugins/kfp/kfp_decorator.py

```diff
@@ -47,6 +47,11 @@ class KfpInternalDecorator(StepDecorator):
       environment set as environment variables and loaded to Metaflow self
       state within task_pre_step.
 
+    image: str
+      Defaults to None, which means default to base image.
```

Comment:
what is "base image"? I believe you're referring to `--base-image` [link](https://github.com/zillow/metaflow/blob/tz/datasets/metaflow/plugins/kfp/kfp_cli.py#L143-L143) which should be said explicitly. 

Let's create another task to cleanup our "--base-image" parameter, and be more aligned with  https://github.com/Netflix/metaflow/issues/489 using `METAFLOW_CONTAINER_IMAGE` and`python flow.py run --with kfp:image=foo`

Reply:
[AIP-5118](https://zbrt.atl.zillow.net/browse/AIP-5119) created.